app/views/drawings/_form.html.haml

```diff
@@ -17,23 +17,25 @@
         .form-group
           = f.input :country, priority: TOP_COUNTRIES, selected: (@drawing.new_record? ? current_user.country : @drawing.country), label: "Country of Origin", include_blank: "Country picture was drawn in", class: 'form-control'
         .form-group
-          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: false, required: true, prompt: "Subject matter of drawing"
+          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: 'Subject matter of drawing', required: true, selected: "Home / Country of origin"
         .form-group
-          = f.input :description, label: false, placeholder: 'The description, e.g. Children playing in the garden, mountains in background'
+          = f.input :description, label: 'Description of Drawing', placeholder: 'e.g. Children playing in the garden, mountains in background'
 
         %hr
 
         %h4 Details of Artist
         .form-group
-          = f.input :age, type: 'number', label: false, required: true, placeholder: 'Age of artist'
+          = f.input :age, type: 'number', label: 'Age of artist', required: true, placeholder: 'e.g. 8'
         .form-group.genders
+          = f.label 'Gender of Artist'
+          %br
           = f.collection_radio_buttons :gender, radio_genders, :first, :last do |b|
             = b.radio_button
             = b.label(class: "btn")
         .form-group
-          = f.input :story, label: false, required: false, placeholder: 'Context / story of the artist, e.g. interesting cultural or psychological notes (optional)'
-        .form-group.text-center
-          = f.input :mood_rating, as: :select, collection: mood_select_box, include_blank: "How would you rate the mood of the picture from 1 (sad/tragic) to 10 (happy/joyful)", label: false
+          = f.input :story, label: 'Context / story of the artist', required: false, placeholder: 'e.g. interesting cultural or psychological notes (optional)'
+        .form-group
+          = f.input :mood_rating, as: :select, collection: mood_select_box, label: 'How would you rate the mood of the picture? On a scale from 1 (sad/tragic) to 10 (happy/joyful)', selected: "5"
```

Comment:
A blank selection is fine - one of the use cases was someone uploading pics only, then someone else updating the data around it afterwards. My 2 suggestions are in my previous comment